tests/php/test-core.php

```diff
@@ -564,6 +564,29 @@ function( $requested_path, $redirected_to, $status_code ) use ( &$redirect_to, &
 		$this->assertTrue( $redirected, 'Expected that /one/?test=true would redirect to /gohere/?test=true but instead redirected to ' . $redirect_to );
 	}
 
+	/**
+	 * Test a redirect rule with a wildcard that shouldn't match.
+	 */
+	public function testNoRedirectWildcard() {
+		$_SERVER['REQUEST_URI'] = '/one-page/';
+		$redirected             = false;
+		$redirect_to            = '/gohere';
+
+		// Create redirect for testing.
+		srm_create_redirect( '/one/*', $redirect_to );
+		add_action(
+			'srm_do_redirect',
+			function( $requested_path, $redirected_to, $status_code ) use ( &$redirect_to, &$redirected, &$expected ) {
```

Comment:
Can you explain why you pass `$redirect_to` and `$expected` here? I don't see the usage of these variables.

Reply:
@amyevans are you able to explain those variables?

Reply:
Good catch, they aren't used, just relics from copy/pasting another test as a starting point. I'll push up a new commit removing them.